src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -50,6 +50,7 @@
     public static final int    RUBY_REVISION = 0;
     public static final String COMPILE_DATE = "2017";
     public static final String ENGINE = "truffleruby";
+    public static final String ENGINE_VERSION = System.getProperty("graal.version", "unknown");
```

Comment:
I think the version should start with `[0-9].`, like the previous TRUFFLERUBY_VERSION.